### ISSUE TITLE
Support for Coinbase Pro

### ISSUE THREAD

**Issue by Steve0212a** [dougdellolio/gdax-csharp#139]
Do you plan on supporting Coinbase Pro?  If so, will it be supported with this package or a new package?

Thanks.

**Comment by dougdellolio** [comment]
@Steve0212a yes. the rebranding is going happen on June 29 so I would like to switch it over then. However, all previous nuget packages will still be available

**Comment by chrisw000** [comment]
Just to add.... the docs say api.gdax.com will continue to work until 31 December 2018, so current/previous packages will keep working until then.
